test/e2e/scale.go

```diff
@@ -572,10 +610,16 @@ func deleteClusterAndWaitWorker(ctx context.Context, inputChan <-chan string, re
 					}
 				}()
 
+				// Calculate namespace.
+				namespaceName := defaultNamespace
+				if deployClusterInSeparateNamespaces {
+					namespaceName = clusterName
+				}
+
 				cluster := &clusterv1.Cluster{
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      clusterName,
-						Namespace: namespace,
+						Namespace: namespaceName,
 					},
 				}
 				framework.DeleteCluster(ctx, framework.DeleteClusterInput{
```

Comment:
Could we add deletion of the namespace to the cleanup of this test?

Reply:
Done